### DISCUSSION TITLE
MQTT AI Human Detection, Sound detection , Baby crying detection?

### DISCUSSION THREAD

**Discussion by scramatte** [alienatedsec/yi-hack-v5#260] [Q&A]
Hello,

Which camera model is supposed to support MQTT AI Human Detection, Sound detection , Baby crying detection? 
I've tried with Yi-Dome 1080p that has support for baby crying in Yi home  app but I haven't been able to test it yet with MQTT.

About AI Human Detection, Sound detection someone has been able to get them working?

Regards

**Comment by alienatedsec** [comment]
@scramatte I am working on another pre-release to fix the MQTT issues.

> About AI Human Detection, Sound detection someone has been able to get them working?

1080p models (this hack) for baby crying
Allwinner cameras (not this hack) for AI Human Detection and Sound Detection.
